### DISCUSSION TITLE
Hi everybody! Docker compose documentation outdated?

### DISCUSSION THREAD

**Discussion by r3c4ll** [GaetanF/cyborgbackup#66] [Q&A] [answered]
Hi, happy Christmas to all...

I just was testing the docker installation procedure as described at [https://cyborgbackup.readthedocs.io/en/latest/user/install.html#docker-install](https://cyborgbackup.readthedocs.io/en/latest/user/install.html#docker-install).

These are the result containers after following all the process:

```bash
CONTAINER ID   NAME                               CPU %     MEM USAGE / LIMIT     MEM %     NET I/O           BLOCK I/O         PIDS
6269f31dc8cf   cyborgbackup_channel_workers_1     0.05%     79.14MiB / 7.736GiB   1.00%     12.2kB / 4.98kB   4.96MB / 1.21MB   1
a05bf1e284fc   cyborgbackup_celery_1              0.14%     100.6MiB / 7.736GiB   1.27%     818kB / 1.02MB    13.3MB / 1.34MB   2
062af07f6206   cyborgbackup_web_1                 12.08%    196.6MiB / 7.736GiB   2.48%     183kB / 205kB     193MB / 1.31MB    7
42f88f506189   cyborgbackup_celery_beat_1         0.00%     84.49MiB / 7.736GiB   1.07%     21.6kB / 101kB    26.1MB / 1.51MB   1
c8d5563a9a6b   cyborgbackup_callback_receiver_1   0.09%     102.6MiB / 7.736GiB   1.30%     11.5kB / 3.74kB   45.6MB / 1.29MB   5
fa8f24e2afda   cyborgbackup_postgres_1            0.00%     52.46MiB / 7.736GiB   0.66%     604kB / 480kB     112MB / 8.37MB    8
3e5912eda91b   cyborgbackup_rabbitmq_1            1.22%     135.5MiB / 7.736GiB   1.71%     715kB / 521kB     182MB / 3.38MB    26
2fb81c7b4b96   cyborgbackup_mongodb_1             0.37%     115.6MiB / 7.736GiB   1.46%     10kB / 0B         181MB / 4.95MB    32
```

But the resulting screen at  my **http://localhost:8000/** is [https://www.imgpaste.net/image/KkLMDq](https://www.imgpaste.net/image/KkLMDq)

If you need to see the logs they are at [https://pastebin.com/bEacVZFU](https://pastebin.com/bEacVZFU)

Is the documentation outdated or missing something? Any light is appreciated I would like to see it in action!

**Comment by GaetanF** [comment, marked as answer]
Hello,

The UI have been excluded from the main backup system. This will permit to manager multiple CyBorgBackup instance from an unique frontend.

You can use https://ui.cyborgbackup.dev or the app on https://github.com/cyborgbackup/cyborgbackup-ui/releases/tag/v0.2.0 to manage the CyBorgBackup instance. 

Be careful, the frontend need to have an access to the web container to connect on the api.

Regards,

**Reply by r3c4ll** [reply]
Thank you for the answer... I'll try it.

> "Be careful, the frontend need to have an access to the web container to connect on the api."

Any specific doc or recommendation about how to connect them?

**Comment by AeliusSaionji** [comment]
It seems to be outdated again: this command refers to a container that doesn't exist?
`docker-compose exec web /bin/bash`
Manually running the following commands doesn't properly create the admin user, perhaps because the password 'admin' is being rejected for simplicity?

I haven't been able to get this up.
